app/src/main/java/com/zulip/android/ZulipActivity.java

```diff
@@ -418,6 +418,7 @@ public void doNarrow(NarrowFilter filter) {
 
     @Override
     public void onNarrow(NarrowFilter narrowFilter) {
+        // TODO: check if already narrowed to this particular stream/subject
```

Comment:
is there a reason for squashing this?  it's a bit more general than just to that particular commit in that it applies to the current code AS-IS as well.